code/__defines/misc.dm

```diff
@@ -70,13 +70,13 @@
 #define ASFX_AMBIENCE	1
 #define ASFX_FOOTSTEPS	2
 #define ASFX_VOTE		4
-#define ASFX_VOX		8
```

Comment:
But... this doesn't refer to vox the species, it means vox as in voice

Reply:
I'm pretty sure it's sheer coincidence, because the toggle says "VOX voice". I'll change it back